src/annotation.js

```diff
@@ -23,6 +23,7 @@ export default {
     chartStates.set(chart, {
       annotations: [],
       elements: [],
+      drawableElements: [],
```

Comment:
I would have called it `visibleElements`, but I think `drawableElements` is actually better name.

Reply:
done!